components-part1/src/components/Logo.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const StyledLogo = styled.div`
+  color: #f65261;
+  height: 24px;
+  width: 151px;
+  margin-left: 20px;
+  transform: translateY(-10px);
+  & .logo_1 {
```

Comment:
`.logo_1` and `.logo_2` have only 1 or two props with different values, could extract those common ones and use them where wanted with `${}` syntext.